src/current/cockroachcloud/cloud-api.md

```diff
@@ -380,10 +420,66 @@ Where:
 
   - `{cluster_id}` is the unique ID of this cluster.
   {{site.data.alerts.callout_info}}
-  The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-a-basic-cluster.md %}).
+  The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-your-cluster.md %}).
   {{site.data.alerts.end}}
   - `{secret_key}` is the secret key for the service account.
-  - `{spend_limit}` is the [maximum amount of money, in US cents, you want to spend per month]({% link cockroachcloud/plan-your-cluster.md %}) on this cluster.
+  - `storage_mib_limit` is the maximum number of Mebibytes of storage that the cluster can have at any time during the month. If this limit is exceeded, then the cluster is throttled, where only one SQL connection is allowed at a time, with the expectation that it is used to delete data to reduce storage usage. It is an error for this value to be zero.
```

Comment:
```suggestion
  - `storage_mib_limit`: Applies only to Basic. the maximum number of MiB of storage that the cluster can consume at any time during the month. If this limit is exceeded, then the cluster is throttled, where only one SQL connection is allowed at a time, with the expectation that it is used to delete data to reduce storage usage. It is an error for this value to be zero.
```
I know you copied what is in the API, but I think we use"MiB" way more often than "Mebibytes" or whatever the word is for GiB in the docs.

Reply:
Used the MiB suggestion and have->use; not mentioning Basic because it's in the Basic section.